### PR TITLE
Remove unused dependency

### PR DESCRIPTION
**Pull request by aledbf**

### CODE REVIEW

File: keepalived-vip/main.go [hunk 1]
```diff
@@ -85,8 +84,6 @@ func main() {
 		glog.Fatalf("Terminating execution: %v", err)
 	}
 
-	proc.StartReaper()
```
Comment (bprashanth) [inline keepalived-vip/main.go]: so this doesn't risk the same zombie leaking problem?

Comment (aledbf) [inline keepalived-vip/main.go]: @bprashanth no because we send the `SIGHUP` signal and keepalived reloads the configuration file.

Comment (bprashanth) [inline keepalived-vip/main.go]: Ah, while we're doing this I noticed the public methods in keepalived.go don't have godocs, could you please populate them (and comment above Reload what you just said)

**Comment by bprashanth** [comment]
LGTM but for the comment nit

**Comment by aledbf** [comment]
ping @bprashanth

**Comment by bprashanth** [comment]
ah didn't realize you had a new commit. LGTM.

**Pull request merged**